src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,11 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
-        return null;
+
+        if (email.equals(users[1])) {
+            return users[1];
+        } else {
+            return null;
+        }
```

Comment:
it's a hardcode, it's not ok, you should go through users array and compare their email with passed email parameter